### ISSUE TITLE
ConnectionException - failed to connect to port 80

### ISSUE THREAD

**Issue by vaughany** [Hi-Folks/lara-lens#1]
This is barely an issue and I'm unsure if I should report it, but fundamentally your Artisan command does throw an exception and so probably should be mentioned.

After installing your package and running `./artisan laralens:diagnostic`, it shows the Configs and Runtime Configs, then dies with the following exception:

```

   Illuminate\Http\Client\ConnectionException 

  cURL error 7: Failed to connect to localhost port 80: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)

  at vendor/laravel/framework/src/Illuminate/Http/Client/PendingRequest.php:504
    500|                         $response->throw();
    501|                     }
    502|                 });
    503|             } catch (ConnectException $e) {
  > 504|                 throw new ConnectionException($e->getMessage(), 0, $e);
    505|             }
    506|         }, $this->retryDelay ?? 100);
    507|     }
    508| 

      +40 vendor frames 
  41  artisan:37
      Illuminate\Foundation\Console\Kernel::handle()
```

I'm currently developing my project and so no website running on port 80 is available. I am however running `./artisan serve` in a different shell, which defaults to port 8080, so perhaps the debug status of the app could be taken into account and the scanned port changed appropriately?

I understand that this project is for deployed projects and not those currently in-development, but thought I would mention the above anyway.

**Comment by roberto-butti** [comment]
Thank you @vaughany . Good point!
I agree, we need to catch the exception and show it in a more "human" way ;)
Stay tuned ...

**Comment by roberto-butti** [comment]
fix released: composer update v0.1.4 catch the connection exception and display the error message.
Let me know if it is ok on your side. I already upgraded my application in prod and i don't see any regression.
I tried to replicate locally your case, defining an incorrect port in APP_URL .

```
composer update
```
https://packagist.org/packages/hi-folks/lara-lens

**Comment by vaughany** [comment]
Looks good to me! No exception, lots of useful data. :+1:
